Modify TOH titles and intro (#34758)
angular/angular: aio/content/tutorial/index.md

<h1 class="no-toc">Tour of Heroes app and tutorial</h1>

<div class="callout is-helpful">
<header>Getting Started</header>

If you're new to Angular, you might want to try the [**Getting Started**](start) quick-start app first.
The Getting Started tutorial covers the same major topics&mdash;components, template syntax, routing, services, and accessing data via HTTP&mdash;in a condensed format, following the most current best practices. It uses a partially-completed StackBlitz project, so that you can make modifications and see the results in real time.

In this tutorial, you build your own app from the ground up, providing experience with the development process as well as a more thorough introduction to basic concepts.

> The **Tour of Heroes app** that you create with this tutorial is used the conceptual basis for many examples in this documentation set. Reading this introduction page provides sufficient context for working with those examples. You do not need to do this tutorial to understand those other examples. The Tour of Heroes tutorial is maintained here for context and continuity.

Reviewer: A few thoughts:

- Remember to give sentences a new line each. 
- I don't know that the average reader or non-native English speaker will know the tech writer term "documentation set". I'd never heard it before working on the docs. I've suggested an alternative should you choose to edit it.
- I deleted "maintained" so it would shift to the active voice with the intention of changing the wording, but then it read ok to me. Does it still convey your intention without "maintained"?

```suggestion
The **Tour of Heroes app** that you create with this tutorial serves as the conceptual basis for many examples throughout the Angular documentation. Reading this introduction page provides sufficient context for working with those examples. You do not need to do this tutorial to understand those other examples. The Tour of Heroes tutorial is here for context and continuity.
```

Author: I think that last sentence was there because they were talking about completely replacing this with the StackBlitz one - but in light of recent discussions, that seems unlikely. StackBlitz doesn't introduce users to the actual dev process the way this type of tutorial does.

<h1 class="no-toc">Tour of Heroes app and tutorial</h1>

<div class="callout is-helpful">
<header>Getting Started</header>

> If you're new to Angular, you might want to try the [**Getting Started**](start) quick-start app first.

Reviewer 2: This paragraph (which has existed for some time) presents some problems. We have two tutorials, and we are basically telling people: "You don't need to do this tutorial, but if you don't at least read it, you won't understand all the other examples in our documentation."

I've created https://github.com/angular/angular/issues/34766 to address this further.

Author: For the moment, I am trying to give someone who lands here an idea of how the two approaches are different, and what they can get out of either or both. Any more band-aid ideas? (...since it will be quite a while until the whole tutorial approach is sorted out and delivered, and new people are picking it up every day.)
